### PR TITLE
[FIX] l10n_mx: Fix 1.25% WH tax configuration

### PR DESCRIPTION
**Pull request by seifgneedy**
The 1.25% WH tax was incorrectly configured, it's an ISR withholding tax, not IVA.

task-4101407

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by malb-odoo — COMMENTED** [review]
Thanks for this ! 
@clbr-odoo Can maybe confirm but like this PR https://github.com/odoo/odoo/pull/177566 this will need an upgrade script in the module l10n_mx and also bump up the version of the module ?

**Comment by clbr-odoo** [comment]
@malb-odoo  Yes, but you'll also have to check that the try_loading does handle correctly the update of unusual fields (tax_exigibility, l10n_mx_tax_type). Just try to update it from the button in the UI :slightly_smiling_face:

**Comment by malb-odoo** [comment]
@oco-odoo Isn't it better to add a new tax and renamed the old one or remove it entirely ?

**Review by malb-odoo — APPROVED** [review]

File: addons/l10n_mx/data/template/account.tax-mx.csv [hunk 1]
```diff
@@ -51,10 +51,10 @@
 "","","","","","","","","","","","","","","","tax","invoice","cuenta216_10_20","-DIOT: Retención","",""
 "","","","","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","","","","tax","refund","cuenta216_10_20","+DIOT: Retención","",""
-"tax18","13","1.25% WH","VAT witholding RESICO 1.25%","Retención IVA(-1.25%)","-1.25","percent","purchase","tax_group_iva_ret_1_25","on_payment","cuenta216_10","","iva","Tasa","0","base","invoice","","","Retención de IVA RESICO 1.25%","Retención IVA(-1.25%)"
-"","","","","","","","","","","","","","","","tax","invoice","cuenta216_10_20","-DIOT: Retención","",""
+"tax4","13","1.25% WH","Witholding of income tax RESICO 1.25%","Retención ISR(-1.25%)","-1.25","percent","purchase","tax_group_iva_isr_ret_1_25","on_invoice","cuenta216_04","","isr","Tasa","0","base","invoice","","","Retención de ISR RESICO 1.25%","Retención ISR(-1.25%)"
```
Comment (clbr-odoo) [inline addons/l10n_mx/data/template/account.tax-mx.csv:54]:
```suggestion
"mx_wh_1_25","13","1.25% WH","Witholding of income tax RESICO 1.25%","Retención ISR(-1.25%)","-1.25","percent","purchase","tax_group_iva_isr_ret_1_25","on_invoice","cuenta216_04","","isr","Tasa","0","base","invoice","","","Retención de ISR RESICO 1.25%","Retención ISR(-1.25%)"

```
I'd use another id (even if it's not very consistent) for safety, this xmlid looks like something we probably removed at some point already, which could cause issues.

**Review by clbr-odoo — CHANGES_REQUESTED** [review]

File: addons/l10n_mx/data/template/account.tax-mx.csv (hunk 1 above)
Comment (seifgneedy) [reply addons/l10n_mx/data/template/account.tax-mx.csv]: That makes sense, updated the PR :)

**Review by clbr-odoo — APPROVED** [review]
Thanks !
@robodoo r+

**Pull request closed without merging**